android/WALT/app/src/main/java/org/chromium/latency/walt/ScreenResponseFragment.java

```diff
@@ -110,9 +114,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         spinner.setAdapter(modeAdapter);
         stopButton.setEnabled(false);
         blackBox.setMovementMethod(new ScrollingMovementMethod());
-        chartLayout = view.findViewById(R.id.chart_layout);
+        brightnessChartLayout = view.findViewById(R.id.brightness_chart_layout);
+        latencyChartLayout = view.findViewById(R.id.latency_chart_layout);
         view.findViewById(R.id.button_close_chart).setOnClickListener(this);
-        chart = (LineChart) view.findViewById(R.id.chart);
+        view.findViewById(R.id.button_close_latency_chart).setOnClickListener(this);
+        brightnessChart = (LineChart) view.findViewById(R.id.chart);
+        latencyChart = (HistogramChart) view.findViewById(R.id.latency_chart);
+        latencyChart.setLabel(W2B_INDEX, "White-to-black Latency");
```

Comment:
Let's drop the "Latency", but add M=median when the test is done. (or in some other place on the chart).